pyquil/quil.py

```diff
@@ -978,13 +956,9 @@ def validate_protoquil(program: Program, quilt: bool = False) -> None:
     :param program: The Quil program to validate.
 
     .. deprecated:: 4.0
-       This function will be removed, use the validate_protoquil and validate_quilt
-       methods on Program instead.
+       This function is a no-op and will be removed in future versions of pyQuil
```

Comment:
I think the whole docstring itself should be updated, otherwise that may be confusing

Reply:
I've done this, along with replace the deprecated docs with a deprecated decorator